content/install-keras-environment-from-scratch.md

```diff
@@ -0,0 +1,174 @@
+
+First step is for you to download **Ubuntu 14.04.4 LTS 64 bits** from the official website: http://www.ubuntu.com/download/desktop. Follow their instructions to run the installer on your computer (from a CD or a USB key).
+
+![Ubuntu Logo](/img/setup-keras/install/01-ubuntu-logo.png)
+
+The installer asks you to choose the language of the OS. Then press the **Continue** button.
+
+![Ubuntu - Welcome](/img/setup-keras/install/02-ubuntu-welcome.png)
+
+Make sure you have enough space on your disk and you are connected to the Internet. Check the checkbox **Download updates while installing**, this will save you some time later. We do not need to install third-party software on our machine. We want to keep it as minimal as possible. Then press the **Continue** button.
+
+![Ubuntu - Preparing to install Ubuntu](/img/setup-keras/install/03-ubuntu-preparing.png)
+
+This part is very specific to your own computer. Make sure to not erase sensitive data. Or if you have a dual boot with another OS, try not to erase anything from the other partitions. Then press the **Install Now** button.
+
+![Ubuntu - Installation type](/img/setup-keras/install/04-ubuntu-install-type.png)
+
+Quick confirmation, you still have time to go back and make changes to your partitioning. Then press the **Continue** button.
+
+![Ubuntu - Write the changes to disks](/img/setup-keras/install/05-ubuntu-erase-disk.png)
+
+Select the city you are located to have the correct timezone. Then press the **Continue** button.
+
+![Ubuntu - Where are you?](/img/setup-keras/install/06-ubuntu-location.png)
+
+Select your keyboard layout. Then press the **Continue** button.
+
+![Ubuntu - Keyboard](/img/setup-keras/install/07-ubuntu-keyboard.png)
+
+Enter your name, the name of your computer, a username and a password. Then press the **Continue** button.
+
+![Ubuntu - User Account](/img/setup-keras/install/08-ubuntu-user-account.png)
+
+Install in progress...
+
+![Ubuntu - Install in progress](/img/setup-keras/install/09-ubuntu-install.png)
+
+You are one reboot away from the Ubuntu desktop. Then press the **Restart Now** button.
+
+![Ubuntu - Restart](/img/setup-keras/install/10-ubuntu-restart.png)
+
+Login using the credentials you've entered in the installation process.
+
+![Ubuntu - Login](/img/setup-keras/install/11-ubuntu-login.png)
+
+Congratulation, you're on the Ubuntu desktop ready to install Keras. You might have some updates to install. Then press the **Install Now** button.
+
+![Ubuntu - Updates](/img/setup-keras/install/12-ubuntu-updates.png)
+
+## Install Keras and its dependencies
+
+First, you must open a **Terminal** and run the script bellow, line by line to install all the requirements of Keras and then Keras itself.
+
+```python
```

Comment:
This is a command line, the language should be specified as `sh`. Also applies to the code below.